### PR TITLE
3DFileViewer: Use ArgsParser

### PR DESCRIPTION
**Pull request by NLstn**
Also fix a bug that ENOENT was not properly handled, so running "3DFileViewer foo" crashed the application

### CODE REVIEW

File: Userland/Applications/3DFileViewer/main.cpp [hunk 1]
```diff
@@ -350,6 +338,16 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
...
 
+    StringView filename;
+
+    Core::ArgsParser args_parser;
+    args_parser.add_positional_argument(filename, "3D model file to open", "path", Core::ArgsParser::Required::No);
+    args_parser.parse(arguments);
+
+    if (filename.is_empty()) {
+        filename = "/home/anon/Documents/3D Models/teapot.obj"sv;
+    }
```
Comment (gmta) [inline Userland/Applications/3DFileViewer/main.cpp:349]: You can leave out these braces for single-line if-blocks.

File: Userland/Applications/3DFileViewer/main.cpp [hunk 2]
```diff
@@ -571,8 +569,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     window->show();
 
-    auto filename = arguments.argc > 1 ? arguments.argv[1] : "/home/anon/Documents/3D Models/teapot.obj";
-    widget->load_path(filename);
+    auto file = FileSystemAccessClient::Client::the().request_file_read_only_approved(window, filename);
+    if (file.is_error()) {
+        if (file.error().code() != ENOENT)
```
Comment (gmta) [inline Userland/Applications/3DFileViewer/main.cpp:574]: If your commit message contains a sentence starting with "Also", that's a good indication that you might need to split the commit up. In this case, I think the `ENOENT` fix should go into a separate commit since it has nothing to do with the `ArgsParser` changes.

**Review by gmta — COMMENTED** [review]
Welcome to the project! :^) two nits:

**Comment by NLstn** [comment]
Thank you @gmta, and thanks for the feedback 👍🏽

**Review by gmta — APPROVED** [review]
